README.md

```diff
@@ -1,33 +1,74 @@
-# `Turborepo` Vite starter
+# CLASS-web
 
-This is an official starter Turborepo.
+This is an implementation of CLASS that runs entirely in the browser.
```

Comment:
Would be nice to spell out CLASS, so a random visitor can get quicker insight what this repo is about.

```suggestion
This is an implementation of  the **C**hemistry **L**and-surface **A**tmosphere **S**oil **S**lab (CLASS) model that runs entirely in the browser.
```